### PR TITLE
Move Cert Manager to operators directory

### PR DESCRIPTION
**Pull request by CathalOConnorRH**
Move cert manager to operators directory and install via gitops as it's required for tekton-results.

### CODE REVIEW

File: images/cluster-setup/install.sh [hunk 1]
```diff
@@ -79,26 +79,26 @@ exit_error() {
...
-            context=$(echo -n "${sub}" | cut -d ',' -f 1)
-            cluster=$(echo -n "${sub}" | cut -d ',' -f 2 | cut -d ':' -f 1)
-            if ! (echo "${clusters[@]}" | grep "${cluster}"); then
-                clusters+=( "${cluster}" )
-                contexts+=( "${context}" )
-                kubeconfigs+=( "${kubeconfig}" )
-                printf "    - %s --- %s --- %s \n" "$cluster" "$context" "$kubeconfig"
-            fi
-        done
+  clusters=()
```
[the remark this answers is not in the document]
Comment (CathalOConnorRH) [reply images/cluster-setup/install.sh:82]: Just formatting so I'll remove these.

File: ckcp/openshift_dev_setup.sh [hunk 2]
```diff
@@ -93,6 +94,7 @@ init() {
   cp "$KUBECONFIG" "$WORK_DIR/credentials/kubeconfig/compute/compute.kubeconfig.base"
 
   KUBECONFIG_KCP="$WORK_DIR/credentials/kubeconfig/kcp/admin.kubeconfig.base"
+  echo export KUBECONFIG_KCP
```
[the remark this answers is not in the document]
Comment (CathalOConnorRH) [reply ckcp/openshift_dev_setup.sh:97]: Sorry, this shouldn't be included. I was testing something else. I'll remove.

File: ckcp/openshift_dev_setup.sh [hunk 3]
```diff
@@ -307,6 +293,8 @@ install_compute() {
     WEBHOOK_SECRET="placeholder_webhook" \
     "$GITOPS_DIR/pac/setup.sh"
 
+  install_ckcp
```
[the remark this answers is not in the document]
Comment (CathalOConnorRH) [reply ckcp/openshift_dev_setup.sh]: Updated to include install_register_compute and reinstate install_ckcp step

File: ckcp/openshift_dev_setup.sh [hunk 4]
```diff
@@ -171,25 +172,6 @@ install_openshift_gitops() {
 }
 
 install_cert_manager() {
```
Comment (Roming22) [inline ckcp/openshift_dev_setup.sh:174]: Rename func to `check_cert_manager`, and reinstate `echo -n "  - openshift-cert-manager-operator: "`.

Comment (CathalOConnorRH) [reply ckcp/openshift_dev_setup.sh]: rename and reinstated.

File: ckcp/openshift_dev_setup.sh [hunk 5]
```diff
@@ -76,7 +77,7 @@ parse_args() {
 }
 
 init() {
-  APP_LIST="openshift-gitops ckcp compute"
+  APP_LIST="openshift-gitops compute ckcp register-compute"
```
Comment (bnallapeta) [inline ckcp/openshift_dev_setup.sh:80]: Don't see any function called 'register-compute'.
Also, I see two other functions install_compute and install_register_compute.
- Please rename install_register_compute to register_compute.
- Need to include install_compute in the APP_LIST too.

**Review by bnallapeta — CHANGES_REQUESTED** [review]
Thanks for this restructuring PR @CathalOConnorRH.
Reg GitOps operator : It was earlier decided that we would have GitOps operator as a pre-requisite for our customers who bring their compute which is documented [here](https://github.com/openshift-pipelines/pipelines-service/blob/main/gitops/README.md?plain=1#L14). However, as ckcp is a complete dev setup, we are including all dependencies ourselves. Would need to revert GitOps back to ckcp.

File: ckcp/openshift_dev_setup.sh (hunk 5 above)
Comment (Roming22) [reply ckcp/openshift_dev_setup.sh:80]: @bnallapeta `main()` is processing APP_LIST and generating func names that it's then executing. So APP_LIST looks good.

**Review by Roming22 — CHANGES_REQUESTED** [review]
Bharath is correct, the openshift-gitops operator is a pre-requisite and therefore should not be under the gitops folder.

File: ckcp/openshift_dev_setup.sh (hunk 5 above)
Comment (CathalOConnorRH) [reply ckcp/openshift_dev_setup.sh:79]: Resolving this as I believe the APP_LIST is ok. Please let me know if it's not.

**Comment by Roming22** [comment]
@CathalOConnorRH I'm afraid a rebase is required.

**Comment by CathalOConnorRH** [comment]
> @CathalOConnorRH I'm afraid a rebase is required.

Rebased now.

**Review by Roming22 — APPROVED** [review]
@bnallapeta PTAL and merge if the PR looks good to you too.

**Review by bnallapeta — APPROVED** [review]
LGTM.

**Pull request merged**